### DISCUSSION TITLE
Relative links to a section of a page does not work

### DISCUSSION THREAD

**Discussion by RaviTella** [community/community#40239] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have the following markdown to navigate to a section in another page 

'(../../#monitoring)'

Every time in click on the rendered link it takes to the top of the page instead of the section. 
It works correctly when I open the rendered link in a new tab.
It also works correctly when I click on the rendered link in preview.

**Comment by radar-blip** [comment]
Have you tried taking out the hashtag `#` ?
That sometimes will take you to top of page.

**Reply by RaviTella** [reply]
@linuxlawson thanks for the response. Your suggestion does not work because when the # is removed it looks for the page with name monitoring instead of section.
